retrofit2-clients/src/main/java/com/palantir/remoting3/retrofit2/ForwardingRetrofitCall.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (c) Copyright 2017 Palantir Technologies Inc. All rights reserved.
+ */
+
+package com.palantir.remoting3.retrofit2;
+
+import java.io.IOException;
+import okhttp3.Request;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+@SuppressWarnings({"checkstyle:noclone", "checkstyle:superclone"})
+abstract class ForwardingRetrofitCall<T> implements retrofit2.Call<T> {
```

Comment:
have you considered making this an interface? and then the consumer implements delegate?

Reply:
Sure thing, I just matched the existing `ForwardingCall` pattern we have for okhttp